variables.md

```diff
@@ -25,6 +25,14 @@ For instance:
 
 It is important to note that **all variables can be used as inputs**. This means that even if the *basic income* can be calculated from other variables, I can, for a given simulation, provide it as an input. Then, if another formula asks for the value of *basic income* for a month, the input value will be returned, and the *basic income* formula **won't be executed.**
 
+### Default values
 
- 
+When OpenFisca is not able to calculate the value of a variable for a requested period, a **default value** is returned.
 
+The default value of a variable is returned:
+  - When the value of an input variables is requested, if this variable has not been set in the input for the requested period.
+    > For instance, let's assume the input variable `student` default value is `False`. If the value of `student` for `2017-09` has **not** been set in the input of the simulation, then computing `student` for `2017-09` will return `False`.
+  - When the value of a variable with formulas is requested, if no formula is defined for the requested period. 
+    > For instance, let's assume the variable `basic_income` has a formula that is defined from `2015-01-01`, and a default value of `0`. Computing `basic_income` for `2014-01-01` will return `0`, while computing `basic_income` for `2015-01-01` will use the formula.
```

Comment:
>For Example : ...
(this is more consistent with the rest of the documentation)

the variable `basic_income`'s formula is defined starting `2015-01-01`, and has a default value of `0`. Computing `basic_income` for `2014-01-01` will return `0`, while computing `basic_income` for `2015-01-01` will use the formula.